webpack.config.js

```diff
@@ -127,6 +127,10 @@ const config = (module.exports = {
           ? ENTERPRISE_SRC_PATH + "/plugins"
           : SRC_PATH + "/lib/noop",
     },
+    fallback: {
+      path: false,
+      crypto: false,
+    },
```

Comment:
# Don't include polyfills for `path` and `crypto`

The title should've probably said: "Try not including polyfills for `path` and `crypto`. After this change, Webpack finally finished the whole build for the first time, but the output is simply a blank page. Many things could've gone wrong so far, but I think we will be forced to finally upgrade `babel` as well :/
 
 ```
[frontend] <s> [webpack.Progress] 100% 
[frontend] 
[frontend] ⚠ ｢wdm｣: Entrypoint app-main 25.3 MiB (23.1 MiB) = 6 assets 25 auxiliary assets
[frontend] Entrypoint app-public 18 MiB (17.2 MiB) = 6 assets 25 auxiliary assets
[frontend] Entrypoint app-embed 18 MiB (17.2 MiB) = 6 assets 25 auxiliary assets
[frontend] Entrypoint styles 556 KiB (1.22 MiB) = vendors-node_modules_css-loader_lib_css-base_js-node_modules_css-loader_lib_url_escape_js-nod-a5171a.hot.bundle.js?42dabef00344531a7f2f 368 KiB styles.hot.bundle.js?503d45e5f8d42a08fe8f 187 KiB styles.d67114e33a9c0b281197.hot-update.js 527 bytes 22 auxiliary assets
[frontend] 
[frontend] WARNING in DefinePlugin
[frontend] Conflicting values for 'process.env.NODE_ENV'
[frontend] 
[frontend] 1 warning has detailed information that is not shown.
[frontend] Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.
[frontend] 
[frontend] webpack 5.37.0 compiled with 1 warning in 2298 ms
[frontend] ℹ ｢wdm｣: Compiled with warnings.
```